ui/package.json

```diff
@@ -30,7 +30,7 @@
     "react-use": "^17.3.2",
     "typescript": "^4.7.4",
     "use-nft": "^0.12.0",
-    "wagmi": "^0.4.2"
+    "wagmi": "0.12.8"
```

Comment:
@johnmark13 Should we use `0.12.8` or `^0.12.8`?

Reply:
that is an auto generated text
https://sentry.io/answers/node-difference-between-tilde-caret/